test_fork utest fails to build with GCC 4.7.2

So there is this HPC cluster where the newest GCC available is 4.7.2, and I have tried to build a fresh dev version of OpenBLAS. If threading is disabled it builds just fine, but with threading enabled the build of one of the utests fail. I suspect this is due to the ancient compilers/libraries/kernel present on the system. Older versions of OpenBLAS did not have this issue.
Log:
```
<snip>
gcc -O2 -DUTEST_CHECK -DSANITY_CHECK -DREFNAME=test_forkf_ -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -fPIC -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DASMNAME=test_fork -DASMFNAME=test_fork_ -DNAME=test_fork_ -DCNAME=test_fork -DCHAR_NAME=\"test_fork_\" -DCHAR_CNAME=\"test_fork\" -I..   -c -o test_fork.o test_fork.c
In file included from test_fork.c:35:0:
/usr/include/sys/wait.h:116:22: error: unknown type name ‘__WAIT_STATUS’
In file included from test_fork.c:35:0:
/usr/include/sys/wait.h:169:23: error: unknown type name ‘__WAIT_STATUS’
/usr/include/sys/wait.h:179:38: error: unknown type name ‘__WAIT_STATUS’
test_fork.c: In function ‘__ctest_fork_safety_run’:
test_fork.c:109:13: warning: implicit declaration of function ‘wait’ [-Wimplicit-function-declaration]
test_fork.c:111:13: warning: implicit declaration of function ‘WEXITSTATUS’ [-Wimplicit-function-declaration]
make[1]: *** [test_fork.o] Error 1
<snip>
```

--- a/utest/test_fork.c
+++ b/utest/test_fork.c
@@ -32,6 +32,7 @@ USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 **********************************************************************************/
 
 #include "openblas_utest.h"
+#include <sys/types.h>
 #include <sys/wait.h>
 #include <cblas.h>
 

